code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -94,128 +101,156 @@
 	cost = 1
 
 /datum/spellbook_entry/mindswap
 	name = "Mindswap"
+	desc = "Allows you to switch bodies with a target next to you. You will both fall asleep when this happens, and it will be quite obvious that you are the target's body if someone watches you do it."
 	spell_type = /obj/effect/proc_holder/spell/pointed/mind_transfer
 	category = "Mobility"
 
 /datum/spellbook_entry/forcewall
 	name = "Force Wall"
+	desc = "Create a magical barrier that only you can pass through."
 	spell_type = /obj/effect/proc_holder/spell/targeted/forcewall
 	category = "Defensive"
 	cost = 1
 
 /datum/spellbook_entry/blink
 	name = "Blink"
+	desc = "Randomly teleports you a short distance."
 	spell_type = /obj/effect/proc_holder/spell/targeted/turf_teleport/blink
 	category = "Mobility"
 
 /datum/spellbook_entry/teleport
 	name = "Teleport"
+	desc = "Teleports you to an area of your selection."
 	spell_type = /obj/effect/proc_holder/spell/targeted/area_teleport/teleport
 	category = "Mobility"
 
 /datum/spellbook_entry/mutate
 	name = "Mutate"
+	desc = "Causes you to turn into a hulk and gain laser vision for a short while."
 	spell_type = /obj/effect/proc_holder/spell/targeted/genetic/mutate
 
 /datum/spellbook_entry/jaunt
 	name = "Ethereal Jaunt"
+	desc = "Turns your form ethereal, temporarily making you invisible and able to pass through walls."
 	spell_type = /obj/effect/proc_holder/spell/targeted/ethereal_jaunt
 	category = "Mobility"
 
 /datum/spellbook_entry/knock
 	name = "Knock"
+	desc = "Opens nearby doors and closets."
 	spell_type = /obj/effect/proc_holder/spell/aoe_turf/knock
 	category = "Mobility"
 	cost = 1
 
 /datum/spellbook_entry/fleshtostone
 	name = "Flesh to Stone"
+	desc = "Charges your hand with the power to turn victims into inert statues for a long period of time."
 	spell_type = /obj/effect/proc_holder/spell/targeted/touch/flesh_to_stone
 
 /datum/spellbook_entry/summonitem
 	name = "Summon Item"
+	desc = "Recalls a previously marked item to your hand from anywhere in the universe."
 	spell_type = /obj/effect/proc_holder/spell/targeted/summonitem
 	category = "Assistance"
 	cost = 1
 
 /datum/spellbook_entry/lichdom
 	name = "Bind Soul"
+	desc = "A dark necromantic pact that can forever bind your soul to an \
+	item of your choosing. So long as both your body and the item remain \
+	intact and on the same plane you can revive from death, though the time \
+	between reincarnations grows steadily with use, along with the weakness \
+	that the new skeleton body will experience upon 'birth'. Note that \
+	becoming a lich destroys all internal organs except the brain."
 	spell_type = /obj/effect/proc_holder/spell/targeted/lichdom
 	category = "Defensive"
 
 /datum/spellbook_entry/teslablast
 	name = "Tesla Blast"
+	desc = "Charge up a tesla arc and release it at a random nearby target! You can move freely while it charges. The arc jumps between targets and can knock them down."
```

Comment:
you should probably replace mentions of the phrase "knock down" in these spell descriptions with mentions of paralysis/stuns/knockdowns, so as to update them to reflect our current combat system